File: week-04/day-01/Student.py
class Student(object):
    def learn(self):
        print("I'm learning!")


    def question(self):
        Teacher.answer(self)


class Teacher(object):
    def teach(self):
        Student.learn(self)


    def answer(self):
        print("Answer!")

teacher_01 =Teacher()
teacher_01.teach()
student_01 = Student()
student_01.question()

File: Csharp-course/week-03/day-03/TeachersStudents/TeachersStudents/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TeachersStudents
{
    class Student
    {
        public Student()
        {

        }

        public void Learn()
        {
            Console.WriteLine("Student learning!");
        }

        public void Question(Teacher teacher)
        {
            teacher.Answer();
        }

    }
}
